config/request-docs.php

```diff
@@ -181,4 +181,20 @@
     ],
 
     'use_factory' => false,
+
+    'rules_order' => [
```

Comment:
May I know why do we need to add this ordering?

If the user want to decided the ordering, can't they define from the `rules` ?

`'page' => 'nullable|integer|min:1',`

Reply:
The definition can only be done through the format that the operator defines in the rule as you posted, but when winning as a team the rules are not always defined in the same order so we don't have a standard and at least in my opinion it looks messy in documents like In the print below, I added this option simply to organize the rules cohesively but there is still the possibility of not passing the ordering and thus returning as defined in the method
![image](https://github.com/rakutentech/laravel-request-docs/assets/38543704/efd7d906-1ec9-40fe-8fd6-ae67ab62dbd1)